


try:
    open("xxxxx")

except Exception as e:
    print(e)